src/containers/MyNdla/arenaQueries.ts

```diff
@@ -164,3 +165,16 @@ export const useArenaTopic = (topicId: number, page: number) => {
   });
   return { arenaTopic: data?.arenaTopic, loading, error };
 };
+
+const arenaRecentTopics = gql`
+  query arenaRecentTopics {
+    arenaRecentTopics {
+      ...ArenaTopicFragment
+    }
+  }
+  ${arenaTopicFragment}
+`;
+
+export const useRecentTopics = () => {
+  return useGraphQuery<GQLArenaRecentTopicsQuery>(arenaRecentTopics);
```

Comment:
```suggestion
    const { data, loading, error } =
    useGraphQuery<GQLArenaRecentTopicsQuery>(arenaRecentTopics);
  return { arenaRecentTopics: data?.arenaRecentTopics, loading, error };
```

for å matche de andre queires